index.html

```diff
@@ -15,14 +16,31 @@
     <header class="header">
       <nav class="header-container">
         <a href="#" class="logo">My logo</a>
-        <img src="images/icon_image/Union.png" alt="Union-btn" class="union" />
+        <img
+          src="images/icon_image/Union.png"
+          alt="Union-btn"
+          class="union mobile"
+        />
+        <div class="desktop">
+          <ul class="header-content">
+            <li class="head-words">
+              <a class="links" href="#portfolio">Portfolio</a>
+            </li>
+            <li class="head-words">
+              <a class="links" href="#about">About</a>
+            </li>
+            <li class="head-words">
+              <a class="links" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
       </nav>
     </header>
 
-    <section class="first-container">
+    <section id="portfolio" class="first-container">
```

Comment:
- [ ] The transition from desktop to mobile for your top section is not smooth. The background image quickly reduces to the 375 px width while the page is still wider. This leaves a section of the page without a background image. Kindly fix this. check the image below for clarification 👍🏾 

![transition](https://github.com/Kwamay/My-Porforlio/assets/99656687/d150d2d2-9700-4872-99f4-42fede95811d)